### PR TITLE
Migrate rootcheck dependencies

### PR DESCRIPTION
**Pull request by lchico**
Related to: https://github.com/wazuh/wazuh/issues/17700  

# Description
This PR includes:

 In database -  get_sqlite_query_result
 In sockets - send_request_socket

necessary changes to give support to the rootcheck migrated tests in the wazuh/wazuh repository. Those tests will use this new structure.

### CODE REVIEW

File: src/wazuh_testing/utils/sockets.py [hunk 1]
```diff
@@ -27,3 +28,18 @@ def delete_sockets(path=None):
...
+        query (str): query request command. For example `agent {agent.id} rootcheck delete`.
+        socket_path (str): by default use wdb socket.
+    Returns:
+        list: Query response data.
+    """
+
+    controller = SocketController(socket_path)
+    controller.send(query, size=True)
+    response = controller.receive(size=True)
+    return response
```
Comment (TomasTurina) [inline src/wazuh_testing/utils/sockets.py:45]: Add new line at the end.

File: src/wazuh_testing/utils/sockets.py [hunk 2]
```diff
@@ -27,3 +28,18 @@ def delete_sockets(path=None):
...
+    Args:
+        query (str): query request command. For example `agent {agent.id} rootcheck delete`.
+        socket_path (str): by default use wdb socket.
+    Returns:
+        list: Query response data.
+    """
+
+    controller = SocketController(socket_path)
+    controller.send(query, size=True)
+    response = controller.receive(size=True)
```
Comment (TomasTurina) [inline src/wazuh_testing/utils/sockets.py:44]: Missing close socket:
```suggestion
    response = controller.receive(size=True)
    controller.close()
```

File: src/wazuh_testing/utils/database.py [hunk 3]
```diff
@@ -84,3 +84,27 @@ def run_sql_script(database_path: Union[os.PathLike, str], script_path: Union[os
     """
     with DatabaseAdministrator(database_path) as db:
         db.execute_script(script_path)
+
+
+def get_sqlite_query_result(db_path: str, query: str) -> List[str]:
```
Comment (TomasTurina) [inline src/wazuh_testing/utils/database.py:89]: You won't need this function (https://github.com/wazuh/wazuh/pull/17939#discussion_r1318888252). Remove it too avoid conflicts with this PR: https://github.com/wazuh/qa-integration-framework/pull/14

**Review by TomasTurina — CHANGES_REQUESTED** [review]

**Comment by lchico** [comment]
Thanks, @TomasTurina for your comment, a great improvement. Ready for review.

**Review by TomasTurina — APPROVED** [review]
LGTM!

**Pull request merged**